packages/grafana-runtime/src/types/i18n.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * Hook type for translation function that takes an ID, default message, and optional values
+ * @returns A function that returns the translated string
+ */
+type UseTranslateHook = () => (id: string, defaultMessage: string, values?: Record<string, unknown>) => string;
+
+/**
+ * Type for children elements in Trans component
+ * Can be either React nodes or an object of values
+ */
+type TransChild = React.ReactNode | Record<string, unknown>;
+
+/**
+ * Props interface for the Trans component used for internationalization
+ * @interface
```

Comment:
Just out of curiosity, is the `@interface` tag necessary in this case? 🤔

Reply:
Good catch, I'll remove it 👍

Reply:
Fixed 👍